lib/galaxy/webapps/tool_shed/controllers/repository.py

```diff
@@ -1466,7 +1466,7 @@ def get_tool_dependencies( self, trans, **kwd ):
         changeset_revision = kwd.get( 'changeset_revision', None )
         repository = repository_util.get_repository_by_name_and_owner( trans.app, name, owner )
         dependencies = repository.get_tool_dependencies( changeset_revision )
-        if dependencies is not None:
+        if len( dependencies ) <= 0:
```

Comment:
I'm not a fan of `if <something that is neither explicitly True nor False>:`, and `<= 0` evaluates to True on an empty dict, which is what I'm looking for.